CalendarApp/BookingRoom/Controllers/PostEventController.cs

```diff
@@ -19,8 +22,17 @@ public PostEventController(BookingService bookingService)
         [HttpPost]
         public HttpResponseMessage EventPost(CalendarEvent eventPost)
         {
-            BookingService.PostEvent(ToEventConverter.ToEvent(eventPost), eventPost.CalendarID);
+            List<Event> conflictedEvents = BookingService.ManagerEvent.CheckEvent(eventPost.CalendarID, ToEventConverter.ToEvent(eventPost), eventPost.ForcedEvent);
+            if (conflictedEvents.Count != 0)
+            {
+                var failRequest = new {status = "Unsuccesful", ConflictedList = conflictedEvents};
+                return Request.CreateResponse(HttpStatusCode.BadRequest, failRequest);
```

Comment:
Maybe we should consider sending HttpStatusCode.OK? We have processed request and returning result of the procession